### ISSUE TITLE
Want a genotyped Multisample VCF

### ISSUE THREAD

**Issue by rmcolq** [rmcolq/pandora#101]
On line https://github.com/rmcolq/pandora/blob/bed2a41f7d9aab36f2b9322852e5ef934a7ee85b/src/compare_main.cpp#L320 in `pandora compare` we run the `genotype` command. An example VCF from a previous `compare` run 
`/nfs/leia/research/iqbal/rff/data/pandora/compare_cmd_test/outdir/GC00000002_17/GC00000002_17.multisample.vcf`
does not have coverage information and hence running the genotype command did not do anything (genotype skips if can't do anything). For comparison, a VCF output from the `pandora map` command looks like this 
`/nfs/leia/research/iqbal/rmcolq/projects/pandora_variation/analysis/cardio_H131800734/genotyping/pandora_genotyped_30X.vcf` and has the coverage information, and so the genotype information.

The coverage information should I think be added  on line https://github.com/rmcolq/pandora/blob/bed2a41f7d9aab36f2b9322852e5ef934a7ee85b/src/compare_main.cpp#L285 which calls this function here https://github.com/rmcolq/pandora/blob/bed2a41f7d9aab36f2b9322852e5ef934a7ee85b/src/localPRG.cpp#L1409

For this to work, there need to be coverages in the kmergraph associated with the pannode.

**Comment by rmcolq** [comment]
EDIT:
1. I got excited that the above outlines a simple solution to the problem of wanting a multisample VCF, BUT IT MISSES a key thing. We want the VCF to be with respect to a best reference for each gene, determined by the samples seen. So we cannot work with a VCF created when looking at a single sample.
2. Earlier we discussed an alternative method which involves storing the coverage information in a multicoloured kmergraph. I will start some more issues for the steps
